vault/identity_store_oidc.go

```diff
@@ -1575,24 +1617,35 @@ func (i *IdentityStore) expireOIDCPublicKeys(ctx context.Context, s logical.Stor
 	return nextExpiration, nil
 }
 
-func (i *IdentityStore) oidcKeyRotation(ctx context.Context, s logical.Storage) (time.Time, error) {
+func (i *IdentityStore) oidcKeyRotation(ctx context.Context, s logical.Storage) (nextRotation time.Time, maxJwksClientCacheDuration time.Duration, err error) {
```

Comment:
I think we can remove the named return values here? They don't appear to be used, and I'm not sure they'd help with readability in a function of this size.